src/components/bestSellers/BestSellersData.js

```diff
@@ -0,0 +1,48 @@
+import { db } from '../../api/firebase'
+import { collection, getDocs } from 'firebase/firestore'
+import { query, orderBy, limit } from "firebase/firestore";
+import { useEffect, useState } from 'react'
+import { BookCard } from '../bookListElement/BookCard';
+import { StyledBestSellersData, StyledH2 } from '../bestSellers/BestSellers.styled'
+import { TextAndButtons } from './BestSellers.styled'
```

Comment:
StyledBestSellersData, StyledH2  z tego samego pliku co TextAndButtons

Reply:
po zmianie katalogów i nazw plików i tak trzeba to było poprawić